.github/container/bump.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
`bump.sh` -> `.github/workflows/scripts/bump-manifest.sh` so that the logic remains in CI?